packages/api/src/external/carequality/document/shared.ts

```diff
@@ -65,6 +65,7 @@ export const cqToFHIR = (
       value: docId,
     },
     docStatus,
+    status: "current",
```

Comment:
This fixes the DocumentReference issue with no status

Reply:
Assuming we won't have the same issue with CW?

Reply:
Yea we set it in CW